packages/web-components/fast-element/docs/api-report.md

```diff
@@ -96,6 +93,13 @@ export abstract class Binding<TSource = any, TReturn = any, TParent = any> {
     policy?: DOMPolicy | undefined;
 }
 
+// @public
+export interface BindingSource {
```

Comment:
Namig nit: based on other terminology around FE, "BindingSource" makes me think this represents the object *provided* to a binding as the source object. I'm struggling a bit to come up with a better suggestion, perhaps `BindingConfig` or `BindingData`? Not a big deal if there isn't another good name, just thought I'd bring up the conflation.

Reply:
I had a hard time coming up with a name I like for that. I'll think a bit more about it and see if anything comes to mind.

Reply:
Maybe just `BindingDirective`.